### PR TITLE
Add compute intensive example

### LINKED ISSUE #1445: Create compute intensive example
Test both cost of compute time and performance wrt to disk operations (few large files vs many small files).

Example needs to be setup such that we can easily measure the impact of the scenario parameters.

### PR DESCRIPTION
**Pull request by frode-aarstad**
Resolves #1445

### CODE REVIEW

File: ert3/evaluator/_evaluator.py [hunk 1]
```diff
@@ -34,6 +34,11 @@ def _prepare_input(ee_config, stages_config, inputs, evaluation_tmp_dir):
     tmp_input_folder = evaluation_tmp_dir / "prep_input_files"
     os.makedirs(tmp_input_folder)
     ee_storage = storage_driver_factory(ee_config["storage"], tmp_input_folder)
+    # record2location = {}
+    # for idx in range(0, len(stages_config)-1):
+    #    record2location.update(
+    #        {input.record: input.location for input in stages_config[idx].input}
+    #    )
     record2location = {input.record: input.location for input in stages_config[0].input}
```
Comment (frode-aarstad) [inline ert3/evaluator/_evaluator.py:42]: Not sure why we always use the input of stage 0  (stages_config[0].input). What if I want to have 2 stages with different input, does that not make sense ?

Comment (mortalisk) [reply ert3/evaluator/_evaluator.py:42]: I don't know why, but a comment a few lines up says that only one stage is supported at the moment.
What is you addition about, and why is it commented out?

File: examples/compute_intensive/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+``` json
+{
+    "hash" : "97c10efe01d5c9c88704a12d361d8429b3a6aa2412290a0773109d5d2d603d5e",
+    "timeout_seconds" : 100,
+    "hash_algo" : "sha256",
+    "plaintext" : "easy"
+}
+```
+
+Its possible to control the runing time of the job by changing the timeout.
```
Comment (mortalisk) [inline examples/compute_intensive/README.md:31]: running

File: ert_shared/ensemble_evaluator/prefect_ensemble/client.py [hunk 3]
```diff
@@ -30,7 +32,7 @@ async def _send(self, msg):
                     self.websocket = await self.get_websocket()
                 await self.websocket.send(msg)
                 return
-            except (ConnectionError, OSError):
+            except (ConnectionError, OSError, ConnectionClosedError):
```
Comment (markusdregi) [inline ert_shared/ensemble_evaluator/prefect_ensemble/client.py:35]: Is this a fix of some kind?

File: examples/compute_intensive/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+## Compute intensive example
+The purpose of this example is to test both cost of compute time and performance of disk operations.
+
+### How to run
+See run_test
+
+``` sh
+rm -fr .ert
```
Comment (markusdregi) [inline examples/compute_intensive/README.md:8]: Remember to remove this. We are soon introducing the possibility to remove experiments and that would be the correct approach if the experiment have already been ran...

File: examples/compute_intensive/run_test [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+rm -fr .ert
```
Comment (markusdregi) [inline examples/compute_intensive/run_test:2]: Remember to remove this. We are soon introducing the possibility to remove experiments and that would be the correct approach if the experiment have already been ran...

File: ert_shared/ensemble_evaluator/prefect_ensemble/client.py (hunk 3 above)
Comment (frode-aarstad) [reply ert_shared/ensemble_evaluator/prefect_ensemble/client.py:35]: Yes. Perhaps we should address this in a separate issue. What do you think @mortalisk.

Comment (markusdregi) [reply ert_shared/ensemble_evaluator/prefect_ensemble/client.py:35]: At least separate it out into a separate commit. But, the preferable option is to make it into a separate PR and get it in faster :)

**Comment by frode-aarstad** [comment]
I think we should just merge this into the POC branch. Then if we find it useful later we can also add it to the master

**Review by markusdregi — APPROVED** [review]
Please rebase on `poc` to loose the extra commits and add the missing new line in three of the files... Then, you are good to go 🚀

**Pull request merged**